skins/oasis/modules/UserPagesHeaderController.class.php

```diff
@@ -71,325 +74,166 @@ public static function getUserName( Title $title, $namespaces, $fallbackToGlobal
 
 	/**
 	 * Get list of links for given username to be shown as tabs
+	 * @param string $userName
+	 * @return array Array of tabs - each tab is an array containing an HTML link, the data-id attribute and whether it is selected
 	 */
 	private function getTabs( $userName ) {
 		wfProfileIn( __METHOD__ );
-		global $wgTitle, $wgEnableWikiaFollowedPages;
 
-		$tabs = array();
-		$namespace = $wgTitle->getNamespace();
+		$tabs = [];
+		$namespace = $this->wg->Title->getNamespace();
 
 		// profile
-		$tabs[] = array(
-				'link' => Wikia::link( Title::newFromText( $userName, NS_USER ), wfMsg( 'profile' ) ),
-				'selected' => ($namespace == NS_USER),
-				'data-id' => 'profile',
-				);
+		$tabs[] = [
+			'link' => Linker::link( Title::newFromText( $userName, NS_USER ), wfMessage( 'profile' )->escaped() ),
+			'selected' => ( $namespace == NS_USER ),
+			'data-id' => 'profile',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "data-id" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)